config/routes.rb

```diff
@@ -4,7 +4,12 @@
   root "users#index"
   get "/users/:id", to: "users#show"
   get "/users/:user_id/posts", to: "posts#index"
+  get "/users/:user_id/all_posts", to: "posts#all_posts"
+  get "/users/:author_id/posts/new", to: "posts#new"
+  post "/users/:author_id/posts/create", to: "posts#create"
   get "/users/:user_id/posts/:id", to: "posts#show"
+  post "/users/:user_id/posts/:post_id/comments", to: "comments#create"
```

Comment:
- [ ] With the new and create actions method above in the comments controller, you'll now required a pair of routes for creating comments.

```ruby
 POST   /posts/:post_id/comments(.:format)     comments#create
 GET   /posts/:post_id/comments/new(.:format)     comments#new
```
`
create` is present 👍  , `new` is left 👎